src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchSettingsProvider.cs

```diff
@@ -894,7 +894,7 @@ internal async Task<string> GetLaunchSettingsFilePathNoCacheAsync()
             // see: https://github.com/dotnet/project-system/issues/2316.
 
             // Default to the project directory if we're not able to get the AppDesigner folder.
-            string folder = Path.GetDirectoryName(_commonProjectServices.Project.FullPath);
+            string folder = _commonProjectServices.Project.GetProjectDirectory();
```

Comment:
Does the new `GetProjectDirectory()` from CPS works cross-platform?

Reply:
Yes, it's in the host-agnostic layer so is available outside of VS too.

The CPS PRs were:

- https://devdiv.visualstudio.com/DevDiv/_git/CPS/pullrequest/594043
- https://devdiv.visualstudio.com/DevDiv/_git/CPS/pullrequest/594713